src/utils/BrambleExtensionLoader.js

```diff
@@ -33,7 +33,8 @@ define(function (require, exports, module) {
         "Autosave",
         "brackets-paste-and-indent",
         "BrambleUrlCodeHints",
-        "UploadFiles"
+        "UploadFiles",
+        "bramble-move-file"
```

Comment:
Update the main README file to include this in the list of extensions we load by default.